signer/mar/README.rst

```diff
@@ -0,0 +1,110 @@
+MAR Signing
+===========
+
+.. sectnum::
+.. contents:: Table of Contents
+
+MAR is Firefox's update file format. For a full description, see
+`go.mozilla.org/mar`_.
+
+.. _`go.mozilla.org/mar`: https://godoc.org/go.mozilla.org/mar
+
+
+Configuration
+-------------
+
+The only thing needed to configure a MAR signer is a private key, either RSA or
+ECDSA P256/P384.
+
+.. code:: yaml
+
+	signers:
+    - id: testmar
+      type: mar
+      privatekey: |
+          -----BEGIN PRIVATE KEY-----
+          MIIJQgIBADANBgkqhkiG9w0BAQEFAASCCSwwggkoAgEAAoICAQDHV+bKFLr1p5FR
+		  ...
+          -----END PRIVATE KEY-----
+
+Signature request
+-----------------
+
+This signer supports `/sign/hash`, `/sign/data/` and `/sign/file` endpoints.
```

Comment:
nit: I am mildly offended that `sign/data` is the only one with a trailing slash